### PR TITLE
The user is referred to the wiki on /help

### PR DESCRIPTION
**Pull request by alexanderliney**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -315,6 +315,7 @@ static void print_help(const OptionList &allowed_options)
 {
 	std::cout << _("Allowed options:") << std::endl;
 	print_allowed_options(allowed_options);
+	std::cout << _("\nVisit wiki.minetest.com for more information.") << std_endl;
```
Comment (SmallJoker) [inline src/main.cpp]: *`std::endl`

File: src/main.cpp [hunk 2]
```diff
@@ -145,7 +145,7 @@ int main(int argc, char *argv[])
 	if (!cmd_args_ok
 			|| cmd_args.getFlag("help")
 			|| cmd_args.exists("nonopt1")) {
-		print_help(allowed_options);
```
Comment (SmallJoker) [inline src/main.cpp]: Why this change?

**Comment by DS-Minetest** [comment]
:-1: 
I always saw the `help` command as way to see a list of all commands. I don't want to see always the reference to minetest wiki. Also this shouldn't be shown in game, there the player would more like to see information about the features of the subgame and the mods and this can be done by the server admin, the subgame or the mods by adding an own command.

**Comment by Wuzzy2** [comment]
:-1:
No, this is just a poor excuse for not fixing the problems with low-quality or non-existing in-game help.
Ideally, `/help` or any other in-game feature should be capable of explaining all commands. If this is not possible, then it must be improved.

At the very least, non-trivial core commands should be writen into README files.

**Comment by C1ffisme** [comment]
As much as I would love for the wiki to be an updated, reliable and wealthy source of information on Minetest, I have to agree with @Wuzzy2. Chances are, the wiki will always be slightly out of date, and won't contain any commands that result from mods.

**Comment by dsohler** [comment]
A lot of pages in the wiki are heavily outdated and contain little to no information for advanced functionality (user wiki and dev wiki).

**Comment by paramat** [comment]
:-1: For above reasons.

**Comment by rubenwardy** [comment]
Thank you for the PR, but it doesn't seem to have much support :/
You may get more support by mentioning the wiki in the pause menu.
